README.md

```diff
@@ -1,6 +1,8 @@
 # Omnisend Magento 2.x plugin
 
-**Note**: Starting from July 11 of 2023 this plugin is no longer supported by Omnisend. You are free to use the plugin and its source code “as-is” or make a fork to modify it as needed. Alternatively we encourage you to use [our API](https://api-docs.omnisend.com/) to establish an integration with Omnisend for your use case.
+> **Warning**
+>
+> Starting from July 11 2023, Omnisend no longer supports this plugin. You're free to use the plugin and its source code as it is or fork it and modify it as needed. However, we encourage you to use [our API](https://api-docs.omnisend.com/) to set up an integration with Omnisend for your use case.
```

Comment:
fiy: there are discussions on which date to use:
https://omnisend.slack.com/archives/C02QXTN525S/p1689068809442639?thread_ts=1689064465.569149&cid=C02QXTN525S